src/software/jetson_nano/ansible/playbooks/setup_nano.yml

```diff
@@ -0,0 +1,63 @@
+---
+
+- name: Setting up the jetson Nano
+  hosts: THUNDERBOTS_HOSTS
+
+  tasks:
+
+    - name: check internet connection
+      uri:
+        url: "https://google.com"
+        timeout: 30
+        status_code: 200
+
+    - name: enable passwordless sudo for rsync
+      become: true
+      become_method: sudo
+      lineinfile:
+        path: /etc/sudoers
+        state: present
+        insertafter: EOF
+        line: '{{ ansible_user }} ALL=NOPASSWD:/usr/bin/rsync'
+
+    - name: Sync Setup Script
+      become: true
+      become_method: sudo
+      tags:
+        - sync
+      ansible.posix.synchronize:
+        src: ../../setup_nano.sh
+        dest: ~/
+        recursive: yes
+        copy_links: yes
+
+    - name: run the setup script
+      become_method: sudo
+      become: true
+      command: bash /home/{{ ansible_user }}/setup_nano.sh >& /tmp/setup.log
+      async: 300
+      poll: 0
+      register: result_async
+      tags:
+        - runScript
+      args:
+        chdir: ~/
+
+    - name: Check Setup Status
+      become: true
+      become_method: sudo
+      async_status:
+        jid: "{{ result_async.ansible_job_id }}"
+      register: result
+      ignore_errors: yes
+      until: result.finished
+      retries: 30
+      delay: 10
+
+    - name: Setup Success Msg
+      debug:
+        msg:
+          - "[Robot ID = {{ inventory_hostname }}]"
+          - "--- Setup successful ---"
+      when: "result.rc is defined and result.rc == 0"
+
```

Comment:
i think we should combine `setup_systemd` and `setup_nano`

we can use the filter to just do setup_systemd if it changes

Reply:
also we need to run jetson_clocks here as well

Reply:
combined the files and added run jetson clocks